integrations/access/mattermost/config.go

```diff
@@ -29,13 +29,13 @@ import (
 
 type Config struct {
 	common.BaseConfig
-	Mattermost MattermostConfig
+	Mattermost MattermostConfig `toml:"mattermost"`
 	StatusSink common.StatusSink
 }
 
 type MattermostConfig struct {
 	URL        string   `toml:"url"`
-	Recipients []string `toml:"recipients"`
+	Recipients []string `toml:"recipients"` // optional
```

Comment:
Is this supposed to be `role_to_recipients`? 

https://github.com/gravitational/teleport/blob/4382dac1679e5d29cabb7c2bc6eee3cbbb48d8f7/integrations/access/common/config.go#L44

Reply:
the original toml field is `recipients`, not sure i should change this field b/c of existing configs? (backwards comp)

Reply:
Got it, I don't have any definite suggestion in that case.